package/src/skia/web/Host.ts

```diff
@@ -17,7 +16,7 @@ export abstract class Host {
   }
 }
 
-export abstract class HostObject<T, N extends string>
```

Comment:
Why this extra base class? Could the `ctor` and `fromValue` be implemented directly on the HostObject class?

Reply:
Because you cannot overload static methods in TS, this is the most elegant solution I found.